packages/components/src/base_tooltip/index.module.scss

```diff
@@ -0,0 +1,14 @@
+.tooltip {
+  // Fix to avoid jumping during tooltip recalculation
+  position: absolute;
+  top: 0;
+  left: 0;
+
+  z-index: -1;
+  visibility: hidden;
+
+  &.isActive {
+    z-index: 100;
```

Comment:
I'm not sure we need a z-index that high. To be honest, I am still not quite too sure about how z-index works. We use z-index: 9 in modals. I wouldn't want feature alerts overlapping modals. Can you check if something like `3` or `4` would work as well?

Reply:
Will put 8 for the sake of modals.